clustered/client/src/test/java/org/ehcache/clustered/client/internal/service/ConnectionClosedTest.java

```diff
@@ -120,11 +121,7 @@ public void testCacheOperationThrowsAfterConnectionClosed() throws Exception {
 
     connection.close();
 
-    try {
-      cache.get(1L);
-    } catch (Exception e) {
-      assertThat(e.getCause().getCause(), instanceOf(ConnectionClosedException.class));
-    }
+    assertThat(cache.get(1L), nullValue());
```

Comment:
Why null now? If the connection is closed explicitly by the client, I don't think it should rely on the resilience strategy (if that's the case here)

Reply:
Was explained. It's the connection that is closed, not the CacheManager. So all good.